docs/apigateway.md

```diff
@@ -104,52 +103,61 @@ You have a series of actions to implement your backend for the book club:
 | putBooks    | PUT | updates book details |
 | deleteBooks | DELETE | deletes a book |
 
-Let's create an API for the book club, named `Book Club`, with `/club` as its HTTP URL base path and `books` as its resource.
+Let's create an API for the book club, named `Book Club`, with `/club` as its HTTP URL base path and `books` as its resource and `{isbn}` as a path parameter used to identify a specific book by its ISBN.
+
+When using path parameters, the API must be defined with a response type of `http`, and the path, including the actual path parameter value(s), will be available in the `__ow_path` field of the action's JSON parameter.
```

Comment:
Add something that shows an example of what the actual value of `__ow_path` would be is not clear if it's going to include everything or just the last portion, I just tested this and it looks ls the last portion, 
__For example if the api is call with `https://${APIHOST}:9001/api/${GENERATED_API_ID}/club/books/123-abc` the value of `__ow_path` will be `/club/books/123-abc`__

Reply:
yeah.. i was not sure how to document what's being passed.  and yes, it is *not* the full url path.  i'll try to figure out a way to doc this better